Tests/Packet++Test/main.cpp

```diff
@@ -341,5 +341,7 @@ int main(int argc, char* argv[])
 	PTF_RUN_TEST(WireGuardCreationTest, "wg");
 	PTF_RUN_TEST(WireGuardEditTest, "wg");
 
+	PTF_RUN_TEST(InfiniBandPacketParsing, "ib");
```

Comment:
We should also add a test for packet creation and packet edit, like we have in other layers

Reply:
ok, I have add a new creation tc for IB.